src/components/Circle/index.tsx

```diff
@@ -0,0 +1,17 @@
+import styles from './Circle.module.scss';
+
+type CircleProps = {
+  color: string;
+  small: boolean;
```

Comment:
```suggestion
  small: boolean;
```
사용할 때 small props 사용 안해도 디폴트 값주셔서 에러 안생기나요??